### ISSUE TITLE
ObjectReferenceAttributes bug

### ISSUE THREAD

**Issue by lencioni** [sds/haml-lint#38]
haml-lint 0.7.0 is reporting a problem from ObjectReferenceAttributes on the first line of the following haml:

``` haml
%h3 Test
%p hi
```
